modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -944,7 +968,46 @@ def copy(self):
         -------
         OpExpr
         """
-        return OpExpr(self.op, self.operands.copy(), self._dtype)
+        op = OpExpr(self.op, self.operands.copy(), self._dtype)
+        if pk := getattr(self, "partition_keys", None):
```

Comment:
Do you have a description of what "partition_keys" field means somewhere?

Reply:
Added to the OpExpr attributes doc.